ballerina-to-openapi/build.gradle

```diff
@@ -132,3 +132,27 @@ compileJava {
         classpath = files()
     }
 }
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
```

Comment:
will this publish to GitHub packages or Is there a reason not to publish in GitHub packages also?

Reply:
```
publishing {
    publications {
        mavenJava(MavenPublication) {
            artifact source: createArtifactZip, extension: 'zip'
        }
    }

    repositories {
        maven {
            name = "GitHubPackages"
            url = uri("https://maven.pkg.github.com/ballerina-platform/openapi-tools")
            credentials {
                username = System.getenv("packageUser")
                password = System.getenv("packagePAT")
            }
        }
    }
}
```

This is how `module-ballerina-openapi` got published, Don't I need to do the same in here

Reply:
I think this should work

Reply:
I think this is correct